microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-![Tony the cat](../media/speed-tony.msft.png)
+The source files for the website are at [MicrosoftEdge / Demos > travel-site](https://github.com/MicrosoftEdge/Demos/tree/main/travel-site). 
 
 
 <!-- ====================================================================== -->
-## Step 1: Audit the site
+## Step 1: Set up the website locally
+
+First, set up the website locally so that you can make changes to it later:
+
+1. Get the website's source code locally: [Download or clone the Demos repo](../sample-code/sample-code.md#download-or-clone-the-demos-repo).
+
+1. Open the folder you just downloaded or cloned in Visual Studio Code.
+
+1. In Visual Studio Code, click **View** > **Terminal** or press `Ctrl` + `` ` ``.
+
+   Visual Studio Code displays the source files in the **Explorer** sidebar, and the **Terminal**:
+
+   ![VS Code, now setup with the source code](./images/vscode-setup.png)
+
+1. In the terminal, type `npx http-server` to start a local web server.
+
+1. In Microsoft Edge, go to http://localhost:8080/travel-site to open the website:
+
+   ![The travel website demo in Microsoft Edge](./images/travel-site.png)
+
+
+<!-- ====================================================================== -->
+## Step 2: Audit the site
 
 Whenever you set out to improve the load performance of a site, always start with an audit.
 
 The audit has two important functions:
 
 *  It creates a **baseline** for you to measure subsequent changes against.
 
-*  It gives you **actionable tips** on what changes have the most impact.
+*  It gives you **actionable tips** on what changes improve performance the most.
+
+#### Establish a baseline
+
+The baseline is a record of how the site performed before you made any performance improvements.
 
-### Setup
+1. In Microsoft Edge, open DevTools by pressing `F12` or `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
 
-First, you must set up the site so that you can make changes to it later.
+1. Open the **Lighthouse** tool.  If you don't see the tool in the main toolbar, click the **More tools** (![More tools](../media/more-tools-icon-light-theme.png)) button.
 
-1. [Open the source code for the site](https://glitch.com/edit/#!/tony).  This tab is referred to as the **editor tab**.
+1. Only keep the **Performance** category selected and clear all other categories. And keep all other options set by default. Here's an explanation of the different options:
 
-   ![The editor tab](../media/speed-glitch-tony-server-js.msft.png)
+   *  **Mode**: set this option to **Navigation (default)** to run tests when the webpage loads, or set the option to **Timespan** to run tests during a given period of time, or set the option to **Snapshot** to run tests on the webpage as it appears now.
```

Comment:
would benefit from trying other formatting.  Main idea at the moment:

avoid:
Do x y z if you want to A.
Use pattern order like:
If you want to A, do x y z.

So, flip the wording in this list/section.  Also consider table? (dubious?) for more skimmability.